packages/web/components/Player/Section/PlayerHero.tsx

```diff
@@ -39,9 +40,8 @@ export const PlayerHero: React.FC<Props> = ({ player }) => {
   return (
     <ProfileSection>
       <VStack spacing={8}>
-        <Avatar
-          w={{ base: 32, md: 56 }}
-          h={{ base: 32, md: 56 }}
+        <PlayerAvatar
```

Comment:
You should leave in the `h` and `w` attributes and replace `src` and `name` with `{...{ player }}`.